modules/elements-color-picker/assets/js/editor/commands/apply.js

```diff
@@ -0,0 +1,52 @@
+import CommandBase from 'elementor-api/modules/command-base';
+
+export class Apply extends CommandBase {
+	apply( { value, trigger } ) {
```

Comment:
JSDoc + explain the `trigger`.

Reply:
It looks like you always sends the `trigger`, so, I suggest sending directly the `picker` and the `swatch` instead + remove the `if ( trigger ) {`.

Reply:
Yeah, I send the `trigger` when I use this command from the code, but I left it as an optional parameter in order to make it easier when executing the command directly without a trigger ( e.g. for testing, QA, etc. )